data/json/npcs/talk_tags.json

```diff
@@ -696,7 +816,12 @@
       "Who's there?",
       "Hello?",
       "Look alive!",
-      "<general_danger_h>"
+      "<general_danger_h>",
+      "Fight or flight?",
```

Comment:
Doesn't seem like something someone would say.

Reply:
I disagree.  It's a little obscure, maybe, but as a shorthand for "we running or are we fighting?" it's fine.